pkg/cmd/root/command.go

```diff
@@ -53,7 +53,7 @@ func Execute() {
 	}
 
 	rootCmd.AddCommand(ssh.NewCommand(&factory, ssh.NewOptions(ioStreams)))
-	rootCmd.AddCommand(targetcmd.NewCommand(&factory, targetcmd.NewOptions(ioStreams), targetProvider))
+	rootCmd.AddCommand(targetcmd.NewTargetCommand(&factory, targetcmd.NewTargetOptions(ioStreams), targetProvider))
```

Comment:
```suggestion
	rootCmd.AddCommand(target.NewCmdTarget(&factory, ioStreams, targetProvider))
```

Reply:
The naming for the function which creates the command is `NewCmd...`. The options are created inside this function. First parameter is the factory, second parameter is the ioStreams object. They do not rename the import of the subcommand package.